src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+        for (int i = 2; i < lineSpaced.length; i++) {
+            lineSpaced[1] = lineSpaced[1].concat(" " + lineSpaced[i]);
+        }
+        return lineSpaced;
+    }
+
+    public static void printLineSpaced(String [] line) {
+        for (String a : line) {
+            System.out.print(a + " ");
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        /*String logo = " ____        _        \n"
+                      + "|  _ \\ _   _| | _____ \n"
+                      + "| | | | | | | |/ / _ \\\n"
+                      + "| |_| | |_| |   <  __/\n"
+                      + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);*/
+        System.out.println("Hello! I'm a Robot");
+        System.out.println("What can I do for you?\n");
+        Task[] tasksList = new Task[100];
+        Scanner in = new Scanner(System.in);
+        String line, commandWord, description;
+        int taskListIndex, currTaskNumber;
+        String[] lineSpaced;
+        boolean isChanged = false;
+
+        line = in.nextLine();
+        lineSpaced = parseCommands(line);
+        currTaskNumber = 0;
+        while(!lineSpaced[0].equals("bye")) {
+            commandWord = lineSpaced[0];
+            switch (commandWord) {
+            case "list":
+                printList(tasksList, currTaskNumber);
+                break;
+            case "mark":
+                description = lineSpaced[1];
+                taskListIndex = Integer.parseInt(description) - 1;
+                isChanged = false;
+                if (!tasksList[taskListIndex].getIsDone()) {
+                    tasksList[taskListIndex].markAsDone();
+                    isChanged = true;
+                }
+                printMarked(isChanged, tasksList[taskListIndex].getStatusIcon(), tasksList[taskListIndex].getDescription());
+                break;
+            case "unmark":
+                description = lineSpaced[1];
+                taskListIndex = Integer.parseInt(description) - 1;
+                isChanged = false;
+                if (tasksList[taskListIndex].getIsDone()) {
+                    tasksList[taskListIndex].markAsUndone();
+                    isChanged = true;
+                }
+                printUnmarked(isChanged, tasksList[taskListIndex].getStatusIcon(), tasksList[taskListIndex].getDescription());
+                break;
+            case "Todo":
+                description = lineSpaced[1];
+                tasksList[currTaskNumber] = new ToDo (description);
+                printAdded(tasksList[currTaskNumber], currTaskNumber+1);
+                currTaskNumber++;
+                break;
```

Comment:
I do not see a command case for `Event` or `Deadline`. Could it be missing from this commit? If so, it's fine you can add it in later versions. 👍